dagger-core/src/main/java/com/gotocompany/dagger/core/sink/SinkOrchestrator.java

```diff
@@ -106,6 +106,9 @@ protected Properties getProducerProperties(Configuration configuration) {
             kafkaProducerConfigs.setProperty(Constants.SINK_KAFKA_COMPRESSION_TYPE_KEY, Constants.SINK_KAFKA_COMPRESSION_TYPE_DEFAULT);
             kafkaProducerConfigs.setProperty(Constants.SINK_KAFKA_MAX_REQUEST_SIZE_KEY, Constants.SINK_KAFKA_MAX_REQUEST_SIZE_DEFAULT);
         }
+        String lingerMs = configuration.getString(Constants.SINK_KAFKA_LINGER_MS_KEY, Constants.SINK_KAFKA_LINGER_MS_DEFAULT);
+        kafkaProducerConfigs.setProperty(Constants.SINK_KAFKA_LINGER_MS_CONFIG_KEY, lingerMs);
```

Comment:
Can you add unit test for this config in the shouldSetKafkaProducerConfigurations() test in SinkOrchestratorTest

Reply:
Added